opts/opts.go

```diff
@@ -182,27 +212,33 @@ func ValidateDevice(val string) (string, error) {
 //    [host-dir:]container-path[:rw|ro]
 // It will also validate the mount mode.
 func ValidatePath(val string) (string, error) {
-	return validatePath(val, true)
+	return validatePath(val, volumeMode)
 }
 
-func validatePath(val string, validateMountMode bool) (string, error) {
+func validatePath(val string, validateMode int) (string, error) {
```

Comment:
I think that passing int values to control flow is a bad design decision, but it's just my personal opinion. I'm sure we can figure out how to have a common interface for both validators.